ARTstor.js

```diff
@@ -1,22 +1,25 @@
 {
-	"translatorID": "5278b20c-7c2c-4599-a785-12198ea648bf",
-	"label": "ARTstor",
-	"creator": "Sebastian Karcher",
-	"target": "^https?://library\\.artstor\\.org",
-	"minVersion": "3.1",
-	"maxVersion": "",
```

Comment:
Yes it did eat the backslash in the paren.

What I to escape the character dot  (.), should I put two double back slash, or single one in the code? Single back slash works in Scaffold.